xml_schema/plugin_addon.xsd

```diff
@@ -0,0 +1,168 @@
+          <xs:element ref="description"/>
+          <xs:element ref="disclaimer"/>
+          <xs:element ref="email"/>
+          <xs:element ref="forum"/>
+          <xs:element ref="language"/>
+          <xs:element ref="platform"/>
+          <xs:element ref="summary"/>
+          <xs:element ref="website"/>
+        </xs:choice>
+      </xs:sequence>
+      <xs:attribute name="compile" type="xs:boolean"/>
+      <xs:attribute name="defaultresolution" type="xs:NMTOKEN"/>
+      <xs:attribute name="library"/>
+      <xs:attribute name="locale"/>
+      <xs:attribute name="name" type="xs:NCName"/>
+      <xs:attribute name="point" use="required" type="xs:NCName"/>
+      <xs:attribute name="start"/>
+      <xs:attribute name="type" type="xs:NCName"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="charsets">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element ref="gui"/>
+        <xs:element ref="subtitle"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="gui">
+    <xs:complexType>
+      <xs:simpleContent>
+        <xs:extension base="xs:NCName">
+          <xs:attribute name="unicodefont" type="xs:boolean"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="dvd">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" ref="menu"/>
+        <xs:element ref="audio"/>
+        <xs:element minOccurs="0" ref="subtitle"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="audio" type="xs:NCName"/>
+  <xs:element name="sorttokens">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" ref="token"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="token">
+    <xs:complexType>
+      <xs:simpleContent>
+        <xs:extension base="xs:NCName">
+          <xs:attribute name="separators"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="broken" type="xs:string"/>
+  <xs:element name="description">
+    <xs:complexType mixed="true">
+      <xs:attribute name="lang" use="required" type="xs:NCName"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="disclaimer">
+    <xs:complexType mixed="true">
+      <xs:attribute name="lang"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="email" type="xs:string"/>
+  <xs:element name="forum" type="xs:anyURI"/>
+  <xs:element name="language" type="xs:string"/>
+  <xs:element name="platform" type="xs:string"/>
+  <xs:element name="summary">
+    <xs:complexType mixed="true">
+      <xs:attribute name="lang" use="required" type="xs:NCName"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="news" type="xs:string"/>
+  <xs:element name="website" type="xs:anyURI"/>
+  <xs:element name="licence" type="xs:string"/>
+  <xs:element name="nofanart" type="xs:boolean"/>
```

Comment:
According to AddonManager you're missing `noicon` too: https://github.com/xbmc/xbmc/blob/0b8f23abe8b9e637071107f3c0c8dac8e89b7b9a/xbmc/addons/AddonManager.cpp#L189

Reply:
no, only repo generator adds nofanart and noicon

Reply:
so `nofanart` should go away :)